src/iproute2_sysrepo.c

```diff
@@ -115,9 +143,162 @@ static const struct cmd {
         { "mptcp",	do_mptcp },
         { "ioam",	do_ioam6 },
         { "stats",	do_ipstats },
+        /* bridge cmds, 
```

Comment:
how about using different `cmds[]` for ip, bridge and tc,
and `do_cmd()` will decide which `cmds[]` to use,

the command to be executed will look like:

```
iproute2-sysrepo ip link add...
iproute2-sysrepo bridge link add ...
iproute2-sysrepo tc qdisc add ...
```

keep in mind we will be using yang generator, and for readability we are using the full iproute2 command in the yang extensions, `ip link add` , `tc qdis add` ... etc.

```c
static int do_cmd(const char* argv0 ,const char *argv1, int argc, char **argv)
{
    struct cmd **curr_cmds;
    if (!strcmp(argv0,"ip")
         curr_cmds= ip_cmds;
     if (!strcmp(argv0,"bridge")
         curr_cmds= brdige_cmds;
     if (!strcmp(argv0,"tc")
         curr_cmds= tc_cmds;
    
    const struct cmd *c;
    for (c = curr_cmds; c->cmd; ++c) {
        if (matches(argv0, c->cmd) == 0)
            return -(c->func(argc - 2, argv + 2));
    }
    return EXIT_FAILURE;
}
```
@adaraiseh  @vjardin , what do you think ?

Reply:
yes, looks good to me

Reply:
ok, let's do it in this PR. @adaraiseh

Reply:
No objection.

It would just be nice to start extending the tests and CI with such cases. Can you do it with the PR too ?